RB-2.0 After Effects plug-in updates

fnordware opened the pull request:
More enhancements to the the After Effects and Photoshop plug-ins, mainly making the Invert button available in all modes, not just LUTs.

Is this branch officially 2.0.1 yet?

hodoulp commented:
> Is this branch officially 2.0.1 yet?
No. The version 2.0.1 is out so the branch is now for 2.0.2 .

doug-walker commented on vendor/aftereffects/OpenColorIO_AE_Context.cpp:
```diff
@@ -573,29 +559,39 @@ bool OpenColorIO_AE_Context::Verify(const ArbitraryData *arb_data, const std::st
+    if(invert == OCIO_INVERT_EXACT)
+    {
+        _cpu_processor = _processor->getOptimizedCPUProcessor(OCIO::OPTIMIZATION_LOSSLESS);
+        _gpu_processor = _processor->getOptimizedGPUProcessor(OCIO::OPTIMIZATION_LOSSLESS);
```
The GPU cannot do exact inversion, so this is fine, but will be ignored.

doug-walker commented on vendor/aftereffects/OpenColorIO_AE_UI.cpp:
```diff
@@ -1049,7 +1080,9 @@ static void DoClickMenus(
             else if(arb_data->action == OCIO_ACTION_CONVERT)
             {
-                menu_items = seq_data->context->getInputs();
+                assert(FALSE);
```
Did you intend to leave this here?

doug-walker reviewed (approved):
Thanks for adding the inversion capability!

michdolan commented on vendor/aftereffects/mac/OpenColorIO_AE_Dialogs_Cocoa.mm:
```diff
@@ -223,21 +223,17 @@ bool ColorSpacePopUpMenu(OCIO::ConstConfigRcPtr config, std::string &colorSpace,
-        else
-        {
-            colorSpacePath = [NSString stringWithFormat:@"%s/%s", family, colorSpaceName];
+            assert(config->getFamilySeparator() == '/');
```
This appears to be enforcing the default  `/` family separator. Can this be changed to support any separator?

hodoulp commented:
Hi @fnordware , what is the status of the pull request?

fnordware commented:
@hodoulp Looks like I'll need to find a little time to address the two comments.

fnordware commented:
@hodoulp By a "little time" I didn't intend for it to take me 5 months. Better late than never?

Rebased to pull in stuff that had been added to RB-2.0 in the meantime. Hopefully did the right thing.

Addressed comments by @doug-walker and @michdolan.

Should I be putting these in the RB-2.1 branch instead?

hodoulp commented:
Let me talk with others but I do **not** think the pull request should be merged in `RB-2.0`. That branch is only for critical bug fixes now.

hodoulp reviewed (changes requested):
Wait to know where to merge it.

fnordware commented:
OK, I'll see if I get put together a RB-2.1 pull request today…

fnordware commented:
Never mind, we'll do it in RB-2.1.
